plugins/notifications-backend/src/service/router.ts

```diff
@@ -525,8 +525,11 @@ export async function createRouter(
       let users = [];
 
       if (!recipients || !title) {
-        logger.error(`Invalid notification request received`);
-        throw new InputError(`Invalid notification request received`);
+        const err = `Invalid notification request received: missing ${
+          !title ? 'title' : ''
+        } ${!recipients ? 'recipients' : ''}`;
+        logger.error(err);
+        throw new InputError(err);
```

Comment:
Avoid logging and throwing, at least without significant additional information in the log. Error responses are already logged so this will lead to duplicate logs. I see it's already the case, but it's an opportunity for cleanup 😁

Reply:
Oh, true, will clean this up